executor/join_test.go

```diff
@@ -104,6 +104,7 @@ func (s *testSuite) TestJoin(c *C) {
 		testleak.AfterTest(c)()
 	}()
 	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("set @@tidb_index_lookup_size = 1")
```

Comment:
if index lookup size is 1, some code may be not touched. We can set it as 2 or 3 when testSuite is inited.